tools/hooks/requirements_changed.rb

```diff
@@ -28,7 +28,9 @@ def get_modified_files
 def optionally_run(cmd, file)
   puts "#{file} changed! Shall I run #{cmd[:cmd]}? [Y/n]"
```

Comment:
could we also update this to something like `"Shall I run #{cmd[:cmd]} from #{dir}?"`